webapp/portlet/src/main/webapp/vue-apps/space-settings/components/SpaceSettings.vue

```diff
@@ -26,17 +27,19 @@ export default {
     // add external components
     const externalComponents = extensionRegistry.loadComponents('external-space').map(component => component.componentOptions.componentImpl);
     this.spaceExternalSettings.push(...externalComponents);
-
-    document.addEventListener('addSpaceSettingsExternalComponents', (event) => {
-      if (event && event.detail) {
-        this.spaceExternalSettings.push(event.detail.componentImpl);
-      }
-    });
-
     document.addEventListener('hideSettingsApps', () => this.displayed = true);
     document.addEventListener('showSettingsApps', () => this.displayed = true);
   },
   mounted() {
+    document.addEventListener('component-external-space-settings-updated', (event) => {
```

Comment:
Shouldn't we update the Chat application and adapt it to the usage of the extensionregistry ?

Reply:
@Rdidmohamedelhadi please do not forget the chat change if needed !